recipes/threadpoolctl/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "2.0.0" %}
+
+package:
+  name: threadpoolctl
+  version: {{ version }}
+
+source:
+  url: https://github.com/joblib/threadpoolctl/archive/{{ version }}.tar.gz
+  sha256: 49b987e9e86168eec68d0c706091bd6a8590268abc249b32bcd2a3909d331a04
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script:
+    - {{ PYTHON }} -m flit install
+
+requirements:
+  host:
+    - python
+    - flit
+  run:
+    - python
+
+test:
+  requires:
+    - numpy
+  imports:
+    - threadpoolctl
+  commands:
+    - python -c "import numpy; from threadpoolctl import threadpool_info; threadpool_info()"
+    - python -c "import numpy; from threadpoolctl import threadpool_limits; threadpool_limits(1)"
+    - python -m threadpoolctl -i numpy
+
+about:
+  home: https://github.com/joblib/threadpoolctl
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Python helpers to control the threadpools of native libraries'
+
+  description: |
+    Python helpers to limit the number of threads used in native
```

Comment:
Maybe add `to limit and introspect` as it is also the goal of the helpers?

Reply:
I'll update